demo/src/renderDemo.js

```diff
@@ -54,26 +54,41 @@ function getComponent(name) {
   return registry[name];
 }
 
+function Demo() {
+  const fixedHeight = 600;
+  const [isExpanded, setIsExpanded] = useState(true);
+  function toggleIsExpanded() {
+    setIsExpanded(prev => !prev);
+  }
```

Comment:
Not super important, but I usually use `useReducer` for this pattern.

```suggestion
  const [isExpanded, toggleIsExpanded] = useReducer(v => !v, true);
```